book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(date, person)
+    rental = Rental.new(date, self, person)
+    @rentals << rental
```

Comment:
- [ ] Kindly remove `@rentals << rental` as it is unnecessary. Creating a rental already adds the rental itself to the array in the constructor, remember?